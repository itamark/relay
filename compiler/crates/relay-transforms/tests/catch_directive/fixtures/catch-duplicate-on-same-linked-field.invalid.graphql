# expected-to-throw
query EnabledUserQuery {
  me @catch @catch(to: NULL) {
    id
  }
}
