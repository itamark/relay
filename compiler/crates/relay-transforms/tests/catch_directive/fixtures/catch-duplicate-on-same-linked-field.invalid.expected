==================================== INPUT ====================================
# expected-to-throw
query EnabledUserQuery {
  me @catch @catch(to: NULL) {
    id
  }
}
==================================== ERROR ====================================
✖︎ The directive `@catch` can only be used once at this location.

  catch-duplicate-on-same-linked-field.invalid.graphql:3:13
    2 │ query EnabledUserQuery {
    3 │   me @catch @catch(to: NULL) {
      │             ^^^^^^
    4 │     id

  ℹ︎ previously used here

  catch-duplicate-on-same-linked-field.invalid.graphql:3:6
    2 │ query EnabledUserQuery {
    3 │   me @catch @catch(to: NULL) {
      │      ^^^^^^
    4 │     id
